File: VB/App_Code/GridData.vb
Option Infer On

Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Public Class GridData

    Public Sub New()

    End Sub
    Public Sub New(ByVal employeeID As Integer, ByVal firstName As String, ByVal lastName As String, ByVal birthDate As Date, ByVal title As String, ByVal hireDate As Date, ByVal displayOrder As Integer)
        Me.EmployeeID = employeeID
        Me.FirstName = firstName
        Me.LastName = lastName
        Me.BirthDate = birthDate
        Me.Title = title
        Me.HireDate = hireDate
        Me.DisplayOrder = displayOrder
    End Sub
    Public Property EmployeeID() As Integer
    Public Property FirstName() As String
    Public Property LastName() As String
    Public Property BirthDate() As Date
    Public Property Title() As String
    Public Property HireDate() As Date
    Public Property DisplayOrder() As Integer
    Public ReadOnly Property FullName() As String
        Get
            Return String.Format("{0} {1}", FirstName, LastName)
        End Get
    End Property

End Class

File: CS/App_Code/GridData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

public class GridData {

    public GridData() {

    }
    public GridData(int employeeID, string firstName, string lastName, DateTime birthDate, string title, DateTime hireDate, int displayOrder) {
        EmployeeID = employeeID;
        FirstName = firstName;
        LastName = lastName;
        BirthDate = birthDate;
        Title = title;
        HireDate = hireDate;
        DisplayOrder = displayOrder;
    }
    public int EmployeeID { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public DateTime BirthDate { get; set; }
    public string Title { get; set; }
    public DateTime HireDate { get; set; }
    public int DisplayOrder { get; set; }
    public string FullName { get { return String.Format("{0} {1}", FirstName, LastName); } }

}